### PR TITLE
[1.13.x-blue][BXMSPROD-2057] Updating references to the new kie-ci actions folder

### PR DESCRIPTION
**Pull request by bacciotti**

### CODE REVIEW

**Review by Ginxo — APPROVED** [review]
LGTM

File: .github/workflows/kogito-examples-pr.yml [hunk 1]
```diff
@@ -28,26 +28,26 @@ jobs:
...
       - name: Java and Maven Setup
-        uses: kiegroup/kogito-pipelines/.ci/actions/maven@1.13.x
+        uses: kiegroup/kie-ci/.ci/actions/maven@1.13.x
         with:
           java-version: ${{ matrix.java-version }}
           maven-version: ${{ matrix.maven-version }}
           cache-key-prefix: ${{ runner.os }}-${{ matrix.java-version }}-maven${{ matrix.maven-version }}
       - name: Build Chain
-        uses: kiegroup/kogito-pipelines/.ci/actions/build-chain@1.13.x
+        uses: kiegroup/kie-ci/.ci/actions/build-chain@1.13.x
```
Comment (Ginxo) [inline .github/workflows/kogito-examples-pr.yml]: definition-file not defined here

**Review by Ginxo — CHANGES_REQUESTED** [review]
please check

File: .github/workflows/kogito-examples-pr.yml [hunk 2]
```diff
@@ -28,26 +28,27 @@ jobs:
     name: Maven Build
     steps:
       - name: Clean Disk Space
-        uses: kiegroup/kogito-pipelines/.ci/actions/ubuntu-disk-space@1.13.x
+        uses: kiegroup/kie-ci/.ci/actions/ubuntu-disk-space@1.13.x
```
Comment (lampajr) [inline .github/workflows/kogito-examples-pr.yml]: `1.13.x` branch does not exist on `kie-ci` repo, we should change to `main` as already done on other repos :)

**Review by lampajr — CHANGES_REQUESTED** [review]

**Review by lampajr — APPROVED** [review]

File: .github/workflows/kogito-examples-pr.yml [hunk 3]
```diff
@@ -28,26 +28,27 @@ jobs:
...
           java-version: ${{ matrix.java-version }}
           maven-version: ${{ matrix.maven-version }}
           cache-key-prefix: ${{ runner.os }}-${{ matrix.java-version }}-maven${{ matrix.maven-version }}
       - name: Build Chain
-        uses: kiegroup/kogito-pipelines/.ci/actions/build-chain@1.13.x
+        uses: kiegroup/kie-ci/.ci/actions/build-chain@main
         env:
           BUILD_MVN_OPTS: '-Dproductized'
         with:
+          definition-file: https://raw.githubusercontent.com/${GROUP:kiegroup}/kogito-pipelines/${BRANCH:main}/.ci/pull-request-config.yaml
```
Comment (Ginxo) [inline .github/workflows/kogito-examples-pr.yml:47]: is it main as default or 1.13.x? or even
```
https://raw.githubusercontent.com/kiegroup/kogito-pipelines/%{process.env.GITHUB_BASE_REF.replace(/(\d*)\.(.*)\.(.*)/g, (m, n1, n2, n3) => `${+n1-7}.${n2}.${n3}`)}/.ci/pull-request-config.yaml
```

**Review by Ginxo — CHANGES_REQUESTED** [review]
please check

File: .github/workflows/kogito-examples-pr.yml (hunk 3 above)
Comment (bacciotti) [reply .github/workflows/kogito-examples-pr.yml:47]: @Ginxo IIUC it is taking 1.13.x.

**Review by Ginxo — APPROVED** [review]
LGTM

**Comment by Ginxo** [comment]
please merge once undrafted

**Pull request merged**